evennia/settings_default.py

```diff
@@ -494,8 +494,12 @@
 # Default sizes for client window (in number of characters), if client
 # is not supplying this on its own
 CLIENT_DEFAULT_WIDTH = 78
-CLIENT_DEFAULT_HEIGHT = 45 # telnet standard is 24 but does anyone use such
-                           # low-res displays anymore?
+# telnet standard height is 24; does anyone use such low-res displays anymore?
+CLIENT_DEFAULT_HEIGHT = 45
+# Help output from CmdHELP are wrapped in an EvMore call
```

Comment:
Thanks for pointing that out.  Class CmdHelp, yes.  (Not CmdHELP). 
Or do you mean to say it needs more explanation in the settings comments?